src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -360,7 +360,7 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
     ttlEntryCxxBufs.reserve(footprintLength);
 
     auto addReads = [&ledgerEntryCxxBufs, &ttlEntryCxxBufs, &ltx, &metrics,
-                     &resources, &sorobanConfig,
+                     &resources, &sorobanConfig, &cfg,
```

Comment:
Nit: `cfg` -> `appCfg` or `appConfig` for clarity and consistency.

Reply:
Done.